src/client/Microsoft.Identity.Client/Platforms/netcore/NetCoreDeviceAuthManager.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+
+namespace Microsoft.Identity.Client.Platforms.netcore
+{
+    internal class NetCoreDeviceAuthManager : IDeviceAuthManager
+    {
+        public bool CanHandleDeviceAuthChallenge { get { return false; } }
```

Comment:
I was under the impression that PKAuth should work on .NET Core. If this is not the case, please make this assumption clear and perhaps open a new feauture / bug to add the missing functionality. In general .Net Core is much more important than .Net Classic and UWP, so there needs to be a strong argument for implementing smth on .Net Fx but not on .NET core.

Reply:
Why wouldn't it work in .NET Core?